modules/vmseries/variables.tf

```diff
@@ -120,9 +120,33 @@ variable "img_version" {
   type        = string
 }
 
-variable "name_application_insights" {
-  default     = null
+variable "create_application_insights" {
+  description = "Enable usage of the Application Insights. The default vailue is set to 'true'."
+  default     = true
+  type        = bool
+}
+
+variable "application_insights_name" {
```

Comment:
What do you think about wrapping all configuration options for app_insight/log_analytics into one vairable of map type? All those are related to a single, common purpose in the modules.